tests/foreman/api/test_contentview.py

```diff
@@ -1826,6 +1826,168 @@ def test_check_needs_publish_flag(self, target_sat):
             if content view needs a publish."""
             )
 
+    def test_inc_update_composite_updated_expected_version(
+        self,
+        target_sat,
+        module_org,
+        fake_yum_repos,
+    ):
+        """Perform an Incremental Update, on a latest CV Version contained in a CompositeCV.
+        Incr. Update the CV's Version to add an Erratum to the Composite's latest Version.
+        Ensure only the intended Version was updated, and the Composite,
+        which contains the expected Versions, and now the added erratum.
+
+        :id: c8ab2dea-f51a-423e-ae1d-027fe6d85674
+
+        :setup:
+            1. Create and sync custom yum repositories.
+            2. Create 3 content views and add a single custom repo to each, auto_publish False.
+            3. Publish v1.0 for a ContentView, with no filters applied (child_cvs[0]).
+            4. Publish v1.0 for a ContentView, with an erratum Exclusion Filter applied (child_cvs[1]).
+            5. Publish v1.0 of the Control ContentView, which we do not change after (child_cvs[2]).
+            6. Create a Composite ContentView, add the 3 prior published Versions as components, use latest True.
+            7. Publish v1.0 of the Composite ContentView.
+
+        :steps: GET content_view_versions/incremental_update <content_view_version_environments> <add_content> <propagate_all_composites>
+            1. Invoke the Incremental Update on the child-CV[1] Version in Library, specifying erratum that was previously filtered out.
+            2. flag --propagate-all-composites set to True (incrementally updates the CompositeCV too)
+            3. Successful incremental update, on the Filtered ContentView Version, v1.0->1.1, adding the filtered erratum.
+            4. Composite components updated, expected Versions are present (see ExpectedResults).
+            5. Publish the Unfiltered ContentView again, v2.0 (child_cvs[0]).
```

Comment:
Shouldn't this step actually happen _before_ the incremental update to ensure that v2.0 is not added by the incremental update? (like step 4 at https://github.com/Katello/katello/pull/11411)